### PR TITLE
XIVY-14003 beautify empty detail view

### PR DESCRIPTION
**Pull request by ivy-lgi**
Target:
image

Result:
beautiful-detail-view

### CODE REVIEW

**Comment by ivy-lgi** [comment]
Correct icon is not available yet, need to ask Anaïs.

File: packages/variable-editor/src/components/detail/EmptyDetail.css [hunk 1]
```diff
@@ -0,0 +1,16 @@
+.empty-detail {
+  flex: auto;
+  align-items: center;
+  justify-content: center;
```
Comment (ivy-lli) [inline packages/variable-editor/src/components/detail/EmptyDetail.css:4]: these two are also available as properties on the flex component, if you like :)

**Review by ivy-lli — APPROVED** [review]
Cool :)

**Comment by ivy-lli** [comment]
Maybe this empty view could also be something for the ui-components, as we may could use it also for the other editors :)

**Pull request merged**